ext/js/language/ja/japanese-transforms.js

```diff
@@ -854,8 +1139,22 @@ export const japaneseTransforms = {
                 suffixInflection('來られる', '來る', ['v1'], ['vk']),
             ],
         },
-        'volitional': {
-            name: 'volitional',
+        '-you': {
```

Comment:
there's a lot of cases here where there is no 'y' in the 'you', which is probably why it was originally named 'volitional' instead

Reply:
What do you think of `-(y)ou` instead?

Reply:
I like `volitional` more. It feels more common to talk about this in terms of that vocabulary vs `(y)ou`